app/assets/stylesheets/_category.scss

```diff
@@ -24,3 +24,18 @@
   color: #bfa24b;
   text-decoration: none;
 }
+
+#menu .floaters{
+  width:18%;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered background, border, width<br>Colon after property should be followed by one space